index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fikerte Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/1652c6c81e.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <a href="#"><span>My Logo</span></a>
+        <a href="#"><i class="fas fa-bars"></i></a>
+    </header>
+    <section class="headline">
+        <h2>I'm Fikerte Glad to see you!</h2>
```

Comment:
- [ ] You can change the `h2` tag with `h1` tag. It is good for seo. [What is the H1 Tag and Why it is Important for SEO ](https://www.reliablesoft.net/h1-tag/#:~:text=The%20HTML%20tag%20is%20the%20first%20header%20tag%20visible,rest%20of%20the%20page%20tags.)